### PR TITLE
Add event code for group changes.

### PR DESCRIPTION
**Pull request by ainmosni**
This MR adds the boilerplate to support emiting group change events.

### CODE REVIEW

File: pkg/events/groups.go [hunk 1]
```diff
@@ -77,3 +77,23 @@ func (GroupMemberRemoved) Unmarshal(v []byte) (interface{}, error) {
...
+type GroupFeature struct {
+	Name  string
+	Value string
+}
+
+// GroupFeatureChanged is emitted when a group feature was changed
+type GroupFeatureChanged struct {
+	Executant *user.UserId
+	UserID    string
+	Features  []GroupFeature
```
Comment (kobergj) [inline pkg/events/groups.go:91]: We don't need a timestamp?

**Review by kobergj — APPROVED** [review]

File: pkg/events/groups.go (hunk 1 above)
Comment (ainmosni) [reply pkg/events/groups.go:91]: Good question, I used the `UserFeatureChanged` as a template, and it also does not have a timestamp field.

File: pkg/events/groups.go [hunk 2]
```diff
@@ -77,3 +77,23 @@ func (GroupMemberRemoved) Unmarshal(v []byte) (interface{}, error) {
...
+// GroupFeature represents a group feature
+type GroupFeature struct {
+	Name  string
+	Value string
+}
+
+// GroupFeatureChanged is emitted when a group feature was changed
+type GroupFeatureChanged struct {
+	Executant *user.UserId
+	UserID    string
```
Comment (kobergj) [inline pkg/events/groups.go:90]: No `GroupID`?

Comment (ainmosni) [reply pkg/events/groups.go]: Oops, guess I wasn't awake enough, fixed now.

File: pkg/events/groups.go (hunk 1 above)
Comment (kobergj) [reply pkg/events/groups.go:91]: Oki. Then it is consistent at least :smile:

**Review by kobergj — APPROVED** [review]

**Pull request merged**